src/backend/distributed/worker/worker_data_fetch_protocol.c

```diff
@@ -1149,6 +1154,22 @@ ExecuteRemoteCommand(const char *nodeName, uint32 nodePort, StringInfo queryStri
 Node *
 ParseTreeNode(const char *ddlCommand)
 {
+	Node *parseTreeNode = ParseTreeRawStmt(ddlCommand);
+
+#if (PG_VERSION_NUM >= 100000)
+	parseTreeNode = ((RawStmt *) parseTreeNode)->stmt;
+#endif
+
+	return parseTreeNode;
+}
+
+
+/*
+ * Parses the given DDL command, and returns the tree node for parsed command.
+ */
+Node *
+ParseTreeRawStmt(const char *ddlCommand)
+{
```

Comment:
This code makes me very sad, everytime I look at it :(.  Obviously not this commit's fault.